sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/maskExpressions.scala

```diff
@@ -79,12 +81,14 @@ import org.apache.spark.unsafe.types.UTF8String
 object MaskExpressionBuilder extends ExpressionBuilder {
   override def functionSignature: Option[FunctionSignature] = {
     val strArg = InputParameter("str")
-    val upperCharArg = InputParameter("upperChar", Some(Literal(Mask.MASKED_UPPERCASE)))
-    val lowerCharArg = InputParameter("lowerChar", Some(Literal(Mask.MASKED_LOWERCASE)))
-    val digitCharArg = InputParameter("digitChar", Some(Literal(Mask.MASKED_DIGIT)))
-    val otherCharArg = InputParameter(
-      "otherChar",
-      Some(Literal(Mask.MASKED_IGNORE, StringType)))
+    val upperCharArg = InputParameter("upperChar",
+      Some(Literal(Mask.MASKED_UPPERCASE, SQLConf.get.defaultStringType)))
```

Comment:
nit: we can use `Literal.create`, so that we don't need to convert String to UTF8String ahead.